appcelerator-archive/amp
# Adding super organization and super user

Resolves #1050 

## How to test
```
$ ampmake build
$ alias amp='amp --server 127.0.0.1:50101'
$ amp cluster create --tag=local
$ amp user signup --name user --password password
$ amp user verify
$ amp login --name user --password password
$ amp stack deploy -c examples/stacks/pinger/pinger.yml pinger
$ amp stack ls
127.0.0.1:50101]
ID                                                                NAME    SERVICE  OWNER  OWNER TYPE
db9b08e121a3656c3be0bc361c2cdf0746a9b0121d903738cb83e3c093d84873  pinger  1        user   USER
$ amp user rm su
[127.0.0.1:50101]
Error: user not authorized
$ amp login --name su --password password
$ amp stack rm pinger
[should be successful]
$ amp org ls
[127.0.0.1:50101]
ORGANIZATION   EMAIL                    CREATED ON
so             super@organization.amp   05 May 17 17:18
$ amp user ls
[127.0.0.1:50101]
USERNAME   EMAIL               CREATED ON
su         super@user.amp      05 May 17 17:18
user       bquenin@axway.com   05 May 17 17:18
$ amp user rm user
[127.0.0.1:50101]
user
$ amp user ls
[127.0.0.1:50101]
USERNAME   EMAIL            CREATED ON
su         super@user.amp   05 May 17 17:18
$ amp org rm so
[127.0.0.1:50101]
Error: user not authorized
$ amp user rm su
[127.0.0.1:50101]
Error: organization must have at least one owner
```
